Jenkinsfile

```diff
@@ -22,7 +22,10 @@ pipeline {
             cloud 'zeebe-ci'
             label "zeebe-ci-build_${buildName}"
             defaultContainer 'jnlp'
-            yamlFile '.ci/podSpecs/distribution.yml'
+            yamlFile ".ci/podSpecs/distribution.yml"
+            // allows the pod to remain active for reuse until the configured number of minutes
+            // has passed since the last step was executed on it.
+            idleMinutes 15
```

Comment:
I think this should be removed based on the PR's description - it's not needed and will leave the pod running for 15min after a build finished but due to using the `label "zeebe-ci-build_${buildName}"` only one build will ever use this pod.

Reply:
Ah, I think this is a leftover from rebasing very often :see_no_evil: